test/extended/builds/controllers.go

```diff
@@ -127,7 +144,7 @@ func RunImageChangeTriggerTest(t testingT, clusterAdminBuildClient buildv1client
 		registryHostname = "registry:8080"
 	)
 
-	testutil.SetAdditionalAllowedRegistries(registryHostname)
+	//testutil.SetAdditionalAllowedRegistries(registryHostname)
```

Comment:
do we have a tracker for a follow up?

Reply:
also added to https://bugzilla.redhat.com/show_bug.cgi?id=1731892